textacy/texts.py

```diff
@@ -643,7 +643,10 @@ def add_text(self, text, lang=None, metadata=None):
         doc.corpus = self
         self.docs.append(doc)
         self.n_docs += 1
-        self.n_sents += doc.n_sents
+        try:
+            self.n_sents += doc.n_sents
+        except ValueError:  # no parser loaded, skip it
+            pass
         self.n_tokens += doc.n_tokens
```

Comment:
nope! sentence segmentation (for better or worse) relies on spacy's syntactic parse, while tokenization does not.